Installed/Makefile

```diff
@@ -957,6 +959,9 @@ Installed/Library/FMTools/OpenWith:	ui ansic MAKEAPPL
 foldered: Installed/Library/FMTools/FolderEd
 Installed/Library/FMTools/FolderEd:	ui ansic MAKEAPPL
 
+cvttool: Installed/Library/FMTools/CvtTool

```

Comment:
Wasn't there an entry for this already:
https://github.com/bluewaysw/pcgeos/blob/763f900fbb9d416cfd3d5ad37e364f7b09855d86/Installed/Makefile#L1688

Reply:
Ok, missed it, was removed down there as seen in the diff.